platform/config/amp-dev-redirects.yaml

```diff
@@ -20,4 +21,7 @@
 
 ### Other redirects
 # Put here redirects for pages that have been moved, etc
-# Add new entries in alphabetical order
+# Add new entries in alphabetical order in the format /path/to/old/url: /path/to/new/url
+
+/documentation/examples/interactivity-dynamic-content/autosuggest_form/: /examples/components/amp-autocomplete
```

Comment:
Correct target URL is: `documentation/examples/components/amp-autocomplete`

Reply:
Ah, sorry. Fixed. :)